front/src/common/Map/Layers/KPLabel.ts

```diff
@@ -9,15 +9,17 @@ export default function getKPLabelLayerProps(params: {
   bottomOffset?: number;
   PKFieldName?: string;
   colors: Theme;
-  maxzoom?: number;
   minzoom?: number;
+  maxzoom?: number;
+  anglePropertyName?: string;
 }): OmitLayer<SymbolLayer> {
   const {
     bottomOffset = 2.5,
     colors,
     PKFieldName = 'kp',
-    maxzoom = 24,
+    anglePropertyName = 'angle',
```

Comment:
I think the anglePropertyName prop is always 'angle', so we can remove it and keep 'angle' l38

Reply:
yep!